src/ThreeJsPanel.ts

```diff
@@ -677,7 +700,24 @@ export class ThreeJsPanel {
       }
     }
 
+    // RENDERING
+    // Step 1: Render meshes, e.g. isosurfaces, separately to a render target. (Meshes are all on
+    //   layer 1.) This is necessary to access the depth buffer.
+    this.camera.layers.set(1);
```

Comment:
I'd suggest using a named constant like MESH_LAYER and VOLUME_LAYER so this value can be tracked across files, like in view3d as well

Reply:
Seconding this